lib/ts/recipe/session/claimBaseClasses/booleanClaim.ts

```diff
@@ -0,0 +1,23 @@
+import { Awaitable } from "../../../types";
```

Comment:
Where will emailverification and roles claim be imported from? I was thinking something like
```
import {EmailVerification, isRole} from "supertokens-node/recipe/session/claims";
```

If the above is true, then this file should be in `recipe/session/claims/base/booleanClaim.ts`

Reply:
We will do this in a different PR later.